diktat-rules/src/test/resources/test/chapter1/naming/file/fileNameTest.kt

```diff
@@ -0,0 +1,12 @@
+package org.cqfn.diktat.resources.test.chapter1.naming.file.resources.test.chapter1.naming.file.naming.file
```

Comment:
I do not know why, but for these files git decided that it was deleted and added not renamed